test/image/mocks/violin_box_multiple_widths.json

```diff
@@ -0,0 +1,44 @@
+{
+    "data": [{
+            "type": "violin",
+            "width": 0.4,
+            "name": "width: 0.4",
+            "x": [0, 5, 7, 8],
+            "side": "positive",
+            "line": {
+                "color": "black"
+            },
+            "fillcolor": "#8dd3c7",
+            "opacity": 0.6,
+            "y0": 0.0
+        }, {
+            "type": "violin",
+            "name": "auto",
+            "x": [0, 5, 7, 8],
+            "side": "positive",
+            "line": {
+                "color": "black"
+            },
+            "fillcolor": "#d3c78d",
+            "opacity": 0.6,
+            "y0": 0.1
+        }, {
+            "type": "box",
+            "width": 0.6,
+            "name": "width: 0.6",
+            "x": [0, 5, 7, 8],
+            "side": "positive",
+            "line": {
+                "color": "black"
+            },
+            "fillcolor": "#c78dd3",
+            "opacity": 0.6,
+            "y0": 0.2
+    }],
+    "layout": {
+        "title": "Joyplot - Violin with multiple widths",
```

Comment:
Let's get rid of the name "joyplot" here if possible, in favour of "ridgeplot"

Reply:
Ok done in https://github.com/plotly/plotly.js/pull/3234/commits/c1d6e76e064ec17afcfacee95c4634cbd08c7195